server/src/sessionManagement.mjs

```diff
@@ -23,6 +23,58 @@
 import { logger } from './logger.mjs';
 import fs from 'fs';
 import yaml from 'js-yaml';
+import WebSocket from 'ws';
+
+class SessionWebSocket {
```

Comment:
Websocket and File stream classes are very similar. Is it possible to write one class with common functions (open, write, close) and do the "If WS else File" logic here? It would make the usage much cleaner

Reply:
Change made to include ws and filestream inside of one class